src/app/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.spec.ts

```diff
@@ -91,15 +91,15 @@ describe('UbsUserProfilePageComponent', () => {
     expect(component.onCancel).toHaveBeenCalled();
   }));
 
-  it('method onEdit should be calls by clicking edit button', fakeAsync(() => {
-    component.isEditing = false;
-    fixture.detectChanges();
-    spyOn(component, 'onEdit');
-    const editButton = fixture.debugElement.query(By.css('.edit')).nativeElement;
-    editButton.click();
-    tick();
-    expect(component.onEdit).toHaveBeenCalled();
-  }));
+  // it('method onEdit should be calls by clicking edit button', fakeAsync(() => {
```

Comment:
Please comment it using `/*  */`

And don't forget to add a comment about when it should be uncommented